src/components/movie-card/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './MovieCard.css';
+
+const MovieCard = ({poster_path, title, release_date, genres, id}) => {
+    let genreList;
+    if(genres.length > 0){
+        genreList = genres.join(', ');
+    }
+    return (
+        <article className="filmCard" key={id}>
+            <img src={poster_path} className="card-img-top" alt="..." />
+            <div className="card-body">
+                <span className="filmTitle">{title}</span>
+                <span className="filmYear">{release_date}</span>
+                <p className="card-text">{genreList}</p>
```

Comment:
```suggestion
                {genres.length > 0 && <p className="card-text">{genres.join(", ")}</p>}
```
Instead of creating unnecessary variable, you can do this. Also, it helps the browser to avoid redundant `<p>`, in case some movie does not have a genre at all 😁